Rewritten autoinstallation chapter of client configuration guide (#917)
uyuni-project/uyuni-docs: modules/client-configuration/pages/autoinst-reinstall.adoc

[[reinstalling-registered-systems]]
= Reinstalling Registered Systems

> To reinstall in place an already registered client, you first have to define an autoinstallation distribution and an autoinstallation profile. For details, see xref:client-configuration:autoinst-xxxxx.adoc[] and xref:client-configuration:autoinst-xxxxx.adoc[].

Reviewer: ```suggestion
To reinstall a registered client in-place, you must define an autoinstallation distribution and an autoinstallation profile.
 For more information, see xref:client-configuration:autoinst-xxxxx.adoc[] and xref:client-configuration:autoinst-xxxxx.adoc[].
```

Author: Yeh. It's the curse of not being a native English speaker.

Author: However, "first" is needed as it's only the first step.

> Once this is done, navigate to menu:Systems[Systems List], select the client to reinstall, and select menu:Provisioning[Autoinstallation > Schedule] subtab.
> 
> At this point, select the autoinstallation profile that you prepared, select a proxy if needed, and press btn:[Schedule Autoinstallation and Finish]. If your client is a traditional client, type [command]``mgr_check`` on that client.
> 
> You may follow the progress of the installation in menu:Provisioning[Autoinstallation > Session Status], or simply on the client's screen. You will see the client reboot, and automatically select in the boot menu a new choice called [guimenu]``reinstall-system``.

Reviewer: ```suggestion
When you have defined the autoinstallation profile and distribution, you can perform the reinstallation.
Navigate to menu:Systems[Systems List], select the client to reinstall, and go to the menu:Provisioning[Autoinstallation > Schedule] subtab.
Select the autoinstallation profile that you prepared, select a proxy if needed, and click btn:[Schedule Autoinstallation and Finish]. 
If your client is a traditional client, type [command]``mgr_check`` at the command prompt on the client.
You can monitor progress of the installation by navigating to menu:Provisioning[Autoinstallation > Session Status], or on the client directly.
The client reboots, and in the boot menu selects a new choice called [guimenu]``reinstall-system``.
```

This is probably a procedure, not prose.

Author: Your prose is nice :smile_cat: , but I will indeed make it a procedure.

image::autoinstall_reinstall.png[scaledwidth=80%]

> This being a purely local mechanism, there is no need for PXE booting capabilities on the client.

Reviewer: ```suggestion
Because PXE booting happens entirely on the local system, there is no need for PXE booting capabilities on the client.
```

This sentence is stuck all alone under the image, can we attach it to a para?

Author: Your suggested change, no, no, no! The point is, PXE booting is not involved at all here! :smile_cat:

Is there a way to attach the image to the procedure above ? It's where it belongs to.

[[reinstalling-registered-systems]]
= Reinstalling Registered Systems

> The in-place reinstallation is a purely local mechanism. There is therefore no need for PXE booting capabilities on the client.

Reviewer: ```suggestion
The in-place reinstallation happens entirely on the local client system. 
There is therefore no need for PXE booting capabilities on the client.
```

Author: ok

> To reinstall a registered client in-place, you must first define an autoinstallation distribution and an autoinstallation profile.

Reviewer: ```suggestion
To reinstall a registered client in-place, you must have defined an autoinstallation distribution and an autoinstallation profile.
```

Author: I have a problem with that phrasing you suggest. I would like to describe steps in a procedure: first do this, then do that.

But if you say that something should have already been done in a prior stage, that's a problem, because we have not even explained yet how to do it!

I would like to keep the "first".

Reviewer: https://github.com/uyuni-project/uyuni-docs/wiki/wug#accessibility

> For details, see xref:client-configuration:autoinst-distributions.adoc[Autoinstallable Distributions] and xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].

Reviewer: ```suggestion
For more information, see xref:client-configuration:autoinst-distributions.adoc[] and xref:client-configuration:autoinst-profiles.adoc[].
```

Author: ok, but the "**more** information" is a bit wrong, because we did not provide information yet. "More information" would suggest a complement, but there's nothing to complement.

Reviewer: It's the standard formula: https://github.com/uyuni-project/uyuni-docs/wiki/wug#for-see

When you have defined the autoinstallation profile and distribution, you can perform the reinstallation.

.Procedure: Reinstall an Already Registered Client

> . Navigate to menu:Systems[Systems List], select the client to reinstall, and go to the menu:Provisioning[Autoinstallation > Schedule] subtab.

Reviewer: ```suggestion
. In the {productname} {webui}, navigate to menu:Systems[Systems List], select the client to reinstall, and go to the menu:Provisioning[Autoinstallation > Schedule] subtab.
```

Author: ok
